tilings/strategies/fusion/disjoint_fusion.py

```diff
@@ -0,0 +1,95 @@
+from typing import Iterator, Optional, Tuple
+
+from comb_spec_searcher import DisjointUnionStrategy, StrategyFactory
+from comb_spec_searcher.exception import StrategyDoesNotApply
+from permuta.misc import DIR_NONE
+from tilings import GriddedPerm, Tiling
+from tilings.algorithms import Fusion
+
+
+class DisjointFusionStrategy(DisjointUnionStrategy[Tiling, GriddedPerm]):
+    def __init__(self, row_idx=None, col_idx=None):
+        self.col_idx = col_idx
+        self.row_idx = row_idx
+        super().__init__(
+            ignore_parent=False, inferrable=True, possibly_empty=False, workable=True
+        )
+
+    def fusion_algorithm(self, tiling: Tiling) -> Fusion:
+        return Fusion(tiling, row_idx=self.row_idx, col_idx=self.col_idx, tracked=False)
+
+    def decomposition_function(self, comb_class: Tiling) -> Tuple[Tiling, ...]:
```

Comment:
Note that we originally drew this strategy as A -> (B, C) where A is a 1x2 domino. C is the unfusion of the first column. B is the pointing in the first cell. The strategy here is implemented C -> (A, B). Note also that the strategy can handle larger tilings, in which case you need to point  to multiple cell. See https://discord.com/channels/383038476553158657/383041809980325896/971080895312326736